### PR TITLE
Fix delegatedTo Relationship

### PR DESCRIPTION
**Pull request by JPEWdev**
In practice, the definition of the delegatedTo relationship was flawed. Since it linked one agent to another, it was impossible to determine which delegatedTo relationships applied to which invokedBy relationships.

To solve this, reword the delegatedTo relationship to indicated that the `to` end must point to the specific invokedBy relationships to which it applies.

### CODE REVIEW

**Comment by JPEWdev** [comment]
It should be noted that this is an unorthodox relationship in that it relates an Actor to another Relationship, which IIRC has not been done before. However, I believe in this context is correctly conveys the information required.

**Comment by sbarnum** [comment]
I have no issue with constraining the `to` or `from` of a Relationship to another Relationship. You are correct that this is completely valid to do.

I am finding it hard to follow the scenario and intent here proposed in this PR.
I think this would benefit very well from a simple diagram of an example.

**Comment by JPEWdev** [comment]
Here is an example that hopefully explains it a little better: https://drive.google.com/file/d/1F6MiE48PAuz8uhk8Jy-g2jCOaAdVolFX/view?usp=sharing

**Review by kestewart — APPROVED** [review]
Thanks for providing the diagram Josh, that helped explain it.   Change looks fine to me.
